File: 30DaysOfCode/day02_operators/operators.cs
using System;
using System.Collections.Generic;
using System.IO;

class Solution {
    static void Main(String[] args) {
        double cost = Convert.ToDouble(Console.ReadLine());
        int tip = Convert.ToInt32(Console.ReadLine());
        int tax = Convert.ToInt32(Console.ReadLine());
        
        int total = (int) Math.Round(cost + (0.01 * tip * cost) + (0.01 * tax * cost));
        
        Console.WriteLine("The total meal cost is " + total + " dollars.");
    }
}

File: 30DaysOfCode/day02_operators/operators.java
import java.util.*;
import java.math.*;

public class Arithmetic {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        double mealCost = scan.nextDouble();
        int tip = scan.nextInt();
        int tax = scan.nextInt();
        scan.close();
      
        int total = (int) Math.round(mealCost + (0.01 * tip * mealCost) + (0.01 * tax * mealCost));
      
        System.out.println("The total meal cost is " + total + " dollars.");
    }
}
